packages/vuetify/src/components/VBtn/VBtn.cy.spec.ts

```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress" />
+import { createVuetify } from '@/framework'
```

Comment:
I think if you import from `@/entry-bundler` we should get the correct global styles loaded as well.